test/lit/passes/roundtrip.wast

```diff
@@ -0,0 +1,46 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
+;; RUN: wasm-opt %s -all --roundtrip -S -o - | filecheck %s
```

Comment:
Wouldn't we see the same result without the `--roundtrip`?

Reply:
No, the problem is in the binary reading code. The testcase is in text, so roundtrip is necessary to write and then read.

Reply:
(The text format allows tuples in a direct way and does not lower then during text reading like binary does.)